### PR TITLE
add basic vignette

### PR DESCRIPTION
**Pull request by lwaldron**

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Package: Rnw2Rmd
 Title: Initial conversion from Rnw to Rmd
-Version: 0.99.2
+Version: 0.100.0
```
Comment (mtmorgan) [inline DESCRIPTION]: I don't think this version number makes sense from a Bioconductor perspective? Submissions are 0.99.x, versions always have to go up. Going up from 100 doesn't lead to 99. Suggest version number 0.99.3.

Comment (lwaldron) [reply DESCRIPTION]: Corrected to 0.99.3, thanks @mtmorgan

**Comment by lwaldron** [comment]
Just making sure this doesn't get lost in your notifications @LiNk-NY

File: vignettes/usage.Rmd [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+```{r setup}
+library(Rnw2Rmd)
+```
+
+
+# Usage
+
+We demonstrate usage on the short `useProbeInfo.Rnw` vignette from the `r Biocpkg("annotate")` package:
+```{r, message=FALSE}
+download.file(url = "https://gist.githubusercontent.com/lwaldron/304388a132b29b802f7241c71c90365b/raw/fb10e070fe2040fc7fab4621130909a319d07c20/useProbeInfo.Rnw",
```
Comment (LiNk-NY) [inline vignettes/usage.Rmd]: Would it be better to use the example Rnw in `inst/example/Rnw2Rmd-test.Rnw` ?

**Review by LiNk-NY — CHANGES_REQUESTED** [review]
We might be able to avoid the download with the example Rnw file in the inst folder.

**Comment by LiNk-NY** [comment]
@lwaldron thoughts on this?

**Comment by lwaldron** [comment]
Yes you're right, I'll fix

**Comment by lwaldron** [comment]
Now uses the internal vignette, and uses noquote to keep the display looking like it does in the Rnw/Rmd files.

**Pull request merged**